package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.151",
+  "version": "1.1.154",
```

Comment:
🤷‍♂ it took me a few attempts forgetting this or that and that's why it's 3 version ^